pyramus/src/main/resources/fi/otavanopisto/pyramus/I18N/pyramuslocale.properties

```diff
@@ -712,6 +708,9 @@ modules.searchModules.advancedSearchButton = Search
 
 # Courses
 
+courses.generic.courseModules.label = Course Modules
+#TODO courses.generic.courseModules.labelHelp = VS ja AVAT: Valitaan oppiaineeksi sopiva VS-kurssi, mik�li kurssi soveltuu VS-kurssiksi, vaikka se soveltuisi my�s muihin opintoihin.<br/>Lukio ja peruskoulu: OPS:in mukainen oppiaine.
```

Comment:
Remember to fix or just default to Finnish

Reply:
This is actually a common theme in the en locale, bunch of the Help labels have no translation and they're just in comments. They are mainly just there to guarantee the line number matching.